### PR TITLE
Fix unused code, indentation, f-strings and conditions in scripts/main

### PR DESCRIPTION
**Pull request by younessZMZ**
### Background
This is a simple clean for the scripts/main by removing unused variables, imports, code blocks and exceptions, and fixing the indentations, conditions, f-strings and docstring.

### Changes

- Remove unused imports.
- Fix indentations.
- Remove unused variables.
- Fix docstring (Prints --> Print) following [docstring conventions](https://peps.python.org/pep-0257/).
- Fix  conditions (not x in list --> x not in list) and (amout > 0 and sup > amout  --> 0 < amount < sup).
- `if result is not None` condition in line 450 is always True.
- Replace string + string with f-string.

### PR Quality Checklist
- [x] My pull request is atomic and focuses on a single change.
- [x] I have thoroughly tested my changes with multiple different prompts.
- [x] I have considered potential risks and mitigations for my changes.
- [x] I have documented my changes clearly and comprehensively.
- [x] I have not snuck in any "extra" small tweaks changes.

### CODE REVIEW

**Review by jacobtohahn — APPROVED** [review]
Code changes look good and professional.

File: scripts/main.py [hunk 1]
```diff
@@ -445,16 +442,9 @@ def main():
...
-            full_message_history.append(chat.create_chat_message("system", result))
-            logger.typewriter_log("SYSTEM: ", Fore.YELLOW, result)
-        else:
-            full_message_history.append(
-                chat.create_chat_message(
-                    "system", "Unable to execute command"))
-            logger.typewriter_log("SYSTEM: ", Fore.YELLOW, "Unable to execute command")
+        # Append the result to the message history
+        full_message_history.append(chat.create_chat_message("system", result))
+        logger.typewriter_log("SYSTEM: ", Fore.YELLOW, result)
```
Comment (nponeccop) [inline scripts/main.py:447]: This change doesn't look right. Probably it creeped in during merging

**Review by nponeccop — CHANGES_REQUESTED** [review]
- There is a strange change. Revert it.
- Revert whitespace changes. We do them in separate whitespace fixes once a day to reduce conflcts
- Move the as e removal to a separate PR

**Comment by nponeccop** [comment]
@younessZMZ There are conflicts now. Also see my requested changes. More atomic requests are easier to review, and typos in general deserve a one-liner PR, we love one-liners but you can keep it here I guess as you only linted 1 file.

**Review by sagarishere — CHANGES_REQUESTED** [review]
:no_entry_sign: **REJECT** with following requests:

Kindly make multiple pull requests. One for space removal and other shenanigans, and one each for where you make changes to the code, whether code logic or even syntax change in code.

You changed the code logic as below, that can affect the output. 

```
        # Append the result to the message history
        full_message_history.append(chat.create_chat_message("system", result))
        logger.typewriter_log("SYSTEM: ", Fore.YELLOW, result)
```

Unfortunately so, whole PR request is rejected from my side.

You should make changes in this PR so that it does not affect the code logic and code output.

**Comment by richbeales** [comment]
closed as resolved by #1393

**Pull request closed without merging**